.github/workflows/upload-release-to-plugin-repo.yml

```diff
@@ -194,19 +187,13 @@ jobs:
                   name: changelog trunk
                   path: trunk
 
-            - name: Commit the content changes
-              working-directory: ./trunk
-              run: |
-                  svn st | grep '^?' | awk '{print $2}' | xargs -r svn add
-                  svn st | grep '^!' | awk '{print $2}' | xargs -r svn rm
-                  svn commit -m "Committing version $VERSION" \
-                   --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
-
-            - name: Create the SVN tag
-              working-directory: ./trunk
-              run: |
-                  svn copy "$PLUGIN_REPO_URL/trunk" "$PLUGIN_REPO_URL/tags/$VERSION" -m "Tagging version $VERSION" \
-                   --no-auth-cache --non-interactive  --username "$SVN_USERNAME" --password "$SVN_PASSWORD"
+            - name: Deploy WordPress Plugin
+              uses: 10up/action-wordpress-plugin-deploy@stable
```

Comment:
We probably need to set the [`$VERSION` env var](https://github.com/10up/action-wordpress-plugin-deploy#optional-environment-variables) here.

Reply:
Ah nvm, that's already available from the job's env vars.